packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/trigger-flow-auth-helper.js

```diff
@@ -36,10 +36,16 @@ async function handleTriggers(context, coreAnswers, previouslySaved) {
   // creating array of trigger values
   const values = Object.values(triggers);
 
+  // Auth lambda config for Triggers
+  let authTriggerConnections = [];
   if (triggers) {
     for (let t = 0; t < keys.length; t += 1) {
       const functionName = `${authResourceName}${keys[t]}`;
       const targetPath = `${targetDir}/function/${functionName}/src`;
+      let config = {};
+      config['triggerType'] = keys[t] === 'PreSignup' ? 'PreSignUp' : keys[t];
+      config['lambdaFunctionName'] = functionName;
```

Comment:
the actual key is PreSignUp , but since we are using PreSignup earlier in our Triggers and changing in CFN ejs template.
Thats why I introduced this logic , otherwise this needs to change all over to make this trigger type same.